src/App.svelte

```diff
@@ -51,6 +51,7 @@
 </script>
 
 <Router>
+  <div class="sr-only" id="announce-route" aria-live="polite" />
```

Comment:
Can we try putting the code that updates this in this file and making it more Svelte-y in terms of its store subscriptions?

Reply:
Moved and used `$:` syntax -> [04ac14c](https://github.com/microbit-foundation/ml-trainer/pull/165/commits/04ac14c919387ea9a69d84b4e3e46f38549ef150)